bridge/src/lib.rs

```diff
@@ -0,0 +1,61 @@
+#[macro_use]
+extern crate rutie;
+extern crate lazy_static;
+
+mod connection;
+
+use connection::Connection;
+use rutie::{Module, Object, Symbol, RString, Encoding, AnyObject, AnyException, Exception, VM, Thread};
+
+wrappable_struct!(Connection, ConnectionWrapper, CONNECTION_WRAPPER);
+
+class!(TemporalBridge);
+
+methods!(
+    TemporalBridge,
+    rtself,
+
+    fn create_connection(host: RString) -> AnyObject {
+        let host = host.map_err(|e| VM::raise_ex(e)).unwrap().to_string();
+
+        let result = Thread::call_without_gvl(move || {
+            Connection::connect(host.clone())
```

Comment:
As far as I can tell it is required to transfer the ownership of a string into the closure. I can maybe try to borrow it instead. Will also check out `cargo clippy`

Reply:
Ok, I may have misread. I figured the `move` on the closure automatically did that since the string is not used after it in this, but it's hard to guess at lifetime stuff from just looking at the code, so I am very likely wrong here somehow.